javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+var formValues = document.querySelector('#contact-form');
+
+formValues.addEventListener('submit', preventDefault);
+
+function preventDefault(event) {
+  event.preventDefault();
+  var messageData = {
+    userName: document.querySelector('#user-name').value,
+    email: document.querySelector('#user-email').value,
+    message: document.querySelector('#user-message').value
+  };
+  console.log(messageData);
+}
```

Comment:
Please reset the form:

---

![image](https://user-images.githubusercontent.com/16143771/170355528-339ceb24-50a8-409a-b0b1-861f53102a43.png)